src/Codeception/Extension/MultiDb.php

```diff
@@ -573,11 +573,11 @@ public function dontSeeInDb($table, $criteria)
      *
      * @return array an array of rows ( depending on the $fetchPdoArgs given )
      */
-    public function getFromDb($table, $criteria, $fields = null, $fetchPdoArgs = array(\PDO::FETCH_ASSOC))
+    public function getFromDb($table, $criteria, $limit = 1, $fields = null, $fetchPdoArgs = array(\PDO::FETCH_ASSOC))
```

Comment:
@codemedic I agree that inserting a new argument half way through the parameter list is something we should avoid, so the argument should be moved to the end with a `null` default value, as you say.

I like your suggestion about implementing a specific function for the single record case.  I would probably go for `getSingleRecordFromDb()`, but that's just preference.

It's worth considering implementing equivalent versions of Doctrine's `getSingleResult()` and `getOneOrNullResult()` functions, which throw exceptions if an unexpected number of results are returned.  We could use `LIMIT 2` then count the results to achieve this.